components/src/web-components/input/gs-date-range-selector.stories.ts

```diff
@@ -92,6 +93,50 @@ export const Default: StoryObj<Required<DateRangeSelectorProps>> = {
         </gs-app>`,
 };
 
+export const RenderAttributesInHtmlInsteadOfUsingPropertyExpression: StoryObj<Required<DateRangeSelectorProps>> = {
+    render: (args) =>
+        html` <gs-app lapis="${LAPIS_URL}">
+            <div class="max-w-screen-lg">
+                <gs-date-range-selector
+                    .dateRangeOptions=${args.dateRangeOptions}
```

Comment:
Should we maybe use the way one would use the component in plain html?
```suggestion
                    dateRangeOptions=${JSON.stringify(args.dateRangeOptions)}
```

Reply:
Unfortunately the lit-analyzer complains about that. I don't know how to ignore that lint, that's why I used the `.` version here... Do you have a better suggestion?

Reply:
Hm... Not really. The lit lint does not show in my idea... 
Maybe this is not this PR, but could be just render plain html without using the lit plugin in storybook? But it seems a lot of effort... https://storybook.js.org/docs/get-started/whats-a-story

Reply:
It also doesn't show in my IDE, `npm run lint` shows it.